frontend/src/app/organization/organization.service.ts

```diff
@@ -21,4 +21,8 @@ export class OrganizationService {
   getOrganizationViewUrl(organization: Organization): string {
     return `${environment.djangoRoot}/organizations/${organization.slug}/${organization.id}`;
   }
+
+  getOrganizationCreateViewUrl(): string {
```

Comment:
@radoslawroszkowiak what do You think about such method? In footer we have such static url directly in template?

Isn't it over bloated here?